notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-818_SITCOMTN-083.ipynb

```diff
@@ -4,8 +4,10 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    plt.savefig(data_dir + f"Bump_Test_Following_Errors_{id}.png")
</pre><p>I think you could create new directory/use exist directory</p><p>1) os.makedirs(<em>directory_name_here</em>, exist_ok=True)</p><p>2) fig.savefig(f"directory_name_here/Bump_Test_Following_Errors_{id}.png")</p><p>: either making new director or just save the file there if there is already directory.</p><p>(e.g. directory_name_here can be "plots" or "result_from_SITCOM818", etc).</p><p>note that os is already imported. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>